### PR TITLE
CHANGELOG: Add github URLs for each issue

### PR DESCRIPTION
**Pull request by briancain**
Couldn't find the consul script so I did it in vim instead:

```
%s/\[GH-\(.*\)\]/[[GH-\1](https:\/\/github.com\/hashicorp\/waypoint\/issues\/\1)]/gc
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,18 +6,18 @@
...
 
 FEATURES:
 
 * **GitOps: Poll for changes and automatically run `waypoint up`**. Waypoint
-  can now trigger a full build, deploy, release cycle on changes detected in Git. [GH-1109]
+  can now trigger a full build, deploy, release cycle on changes detected in Git. [[GH-1109](https://github.com/hashicorp/waypoint/issues/1109)]
 * **Runners: Run Waypoint operations remotely**. Runners are standalone processes that
-  run operations such as builds, deploys, etc. remotely. [GH-1167] [GH-1171]
-* **AWS Lambda**: Add support for building and deploying AWS Lambda workloads [GH-1097]
+  run operations such as builds, deploys, etc. remotely. [[GH-1167] [GH-1171](https://github.com/hashicorp/waypoint/issues/1167] [GH-1171)]
```
Comment (onlydole) [inline CHANGELOG.md:16]:
```suggestion
  run operations such as builds, deploys, etc. remotely. [[GH-1167](https://github.com/hashicorp/waypoint/issues/1167)] [[GH-1171](https://github.com/hashicorp/waypoint/issues/1171)]
```

Comment (onlydole) [reply CHANGELOG.md:16]: Minor nit: Update Markdown Links

**Review by onlydole — CHANGES_REQUESTED** [review]
One small nit, otherwise LGTM 👍🏼

File: CHANGELOG.md (hunk 1 above)
Comment (briancain) [reply CHANGELOG.md:16]: Nice catch! Thanks

**Review by onlydole — APPROVED** [review]
LGTM 👍🏼

**Review by mitchellh — APPROVED** [review]

**Pull request merged**